example/spec/parallel/queueGranules/QueueGranulesSpecPostProcessing.input.payload.json

```diff
@@ -0,0 +1,25 @@
+{
+  "granules": [
+    {
+      "granuleId": "MOD09GQ.A2016358.h13v04.006.2016360104606",
+      "dataType": "MOD09GQ",
+      "version": "006",
+      "files": [
+        {
+          "bucket": "cumulus-test-sandbox-internal",
```

Comment:
since these are actually being created, it should be able to run on anyone's stack and these buckets should probably be templated to the user's system bucket.

Reply:
This hard coded bucket causes problems with the afterAll cleanup  via the waitForGranuleAndDelete.

Reply:
Agreed, will sort.

Reply:
682f31248 should sort it, waiting on a build to test.

Reply:
682f31248